pom.xml

```diff
@@ -1395,7 +1395,7 @@
             <dependency>
                 <groupId>com.google.http-client</groupId>
                 <artifactId>google-http-client-jackson2</artifactId>
-                <version>1.21.0</version>
+                <version>1.23.0</version>
```

Comment:
Looks good, but I think we should be syncing the version of `google-http-client-jackson2` with the versions of `google-http-client` and `google-api-client` (at a minimum).  I notice several of these Google dependencies  (in the same region in this file) all are synced on the same 1.21.0 version, and I'm not sure that's an accident.

Reply:
From the notes here, they all seem to have synced versions/releases: https://developers.google.com/api-client-library/java/

Reply:
I agree, I've revised